l10n_it_fiscal_document_type/models/account_fiscal_position.py

```diff
@@ -7,4 +7,22 @@ class AccountFiscalPosition(models.Model):
     fiscal_document_type_id = fields.Many2one(
         'fiscal.document.type',
         string="Fiscal Document Type",
-        readonly=False)
+        readonly=False
+    )
+    is_company_it = fields.Boolean(
+        compute="_compute_is_company_it"
+    )
+
+    def _compute_is_company_it(self):
```

Comment:
As this check is always the same for the  3 models, could it be written only once?

Reply:
I noticed https://github.com/OCA/l10n-italy/pull/2014#issuecomment-752684726 later ;-)